src/ui/widgets/EmbeddedDisplay/opiParser.ts

```diff
@@ -176,6 +194,22 @@ export function opiParseActions(
               (action.description && action.description._text) || undefined
           }
         });
+      } else if (type === OPEN_DISPLAY) {
+        processedActions.actions.push({
+          type: OPEN_DISPLAY,
+          dynamicInfo: {
+            name: action.path._text,
+            location: descriptionToLocation(action),
+            description:
+              (action.description && action.description._text) || undefined,
+            file: {
```

Comment:
As the TODO says, I feel like the new pages aren't connecting to coniql because they don't receive their macros but am unsure how to correct this.